nestkernel/nest_types.h

```diff
@@ -112,7 +112,9 @@ const long_t long_t_max = LONG_MAX;
 const long_t long_t_min = LONG_MIN;
 
 /**
- * only integral consts are compile time
+ * Largest available double value.
+ *
+ * @note Compile-time constant must have integral type, thus we must define double limits as macros.
```

Comment:
This line is longer than 80 characters and Travis will choke on it once it works again (see #380). Could you break it, e.g., after the comma? And could you add the corresponding comment for the minimum?